pkg/specs/template.go

```diff
@@ -17,6 +17,7 @@ type Template struct {
 	Enum     *Enum    `json:"enum,omitempty" yaml:"enum,omitempty"`
 	Repeated Repeated `json:"repeated,omitempty" yaml:"repeated,omitempty"`
 	Message  Message  `json:"message,omitempty" yaml:"message,omitempty"`
+	OneOf    OneOf    `json:"oneOf,omitempty" yaml:"oneOf,omitempty"`
```

Comment:
Does this overlap with the upcoming PR of @Alma-media?

Reply:
well, need to ask Alex. I showed him this change before, maybe he started developing oneof on top of it.

Reply:
It is fine